### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by SharkFall** [bytedance/android-inline-hook#7] [Q&A]
shadowhook Version
1.0.3

Target Sdk Version
32

Android ABIs
armeabi-v7a, arm64-v8a, x86

Device Manufacturers and Models
pc

Describe the Bug
Could not find a package configuration file provided by "shadowhook" with
  any of the following names:

    shadowhookConfig.cmake
    shadowhook-config.cmake

  Add the installation prefix of "shadowhook" to CMAKE_PREFIX_PATH or set
  "shadowhook_DIR" to a directory containing one of the above files.  If
  "shadowhook" provides a separate development package or SDK, be sure it has
  been installed.

**Comment by BlackBoxing** [comment]
Me too.

**Comment by BlackBoxing** [comment]
I solved the problem by erase x86 architecture unsupported by android-inline-hook from build.gradle.

**Reply by SharkFall** [reply]
oh thanks,
